jws.go

```diff
@@ -235,3 +235,20 @@ func (obj JsonWebSignature) FullSerialize() string {
 
 	return string(mustSerializeJSON(raw))
 }
+
+// MarshalJSON serializes the JWS to JSON.
+func (obj JsonWebSignature) MarshalJSON() (result []byte, err error) {
+	return []byte(obj.FullSerialize()), nil
```

Comment:
Wasn't a point, I guess just an observation, but MarshalJSON has to return err so it's got to be nil.